pkg/cluster/acsk/acsk.go

```diff
@@ -20,10 +20,9 @@ import (
 
 // NodePool describes Alibaba's node fields of a CreateCluster/Update request
 type NodePool struct {
-	InstanceType       string `json:"instanceType"`
-	SystemDiskCategory string `json:"systemDiskCategory,omitempty"`
-	SystemDiskSize     int    `json:"systemDiskSize,omitempty"`
-	Count              int    `json:"count"`
```

Comment:
is autoscaling mandatory? we have both `{Min,Max}Count` and `Count` fields in other providers' nodepool definitions while `Autoscaling` is used as a discriminator.

Reply:
I added the count field in commit "d5623db" but we don't need to get this information from the user, because Alibaba API just requires min and max counts.  AutoScaling is not enabled yet.